CIP-0030/README.md

```diff
@@ -189,6 +201,10 @@ A name for the wallet which can be used inside of the dApp for the purpose of as
 
 A URI image (e.g. data URI base64 or other) for img src for the wallet which can be used inside of the dApp for the purpose of asking the user which wallet they would like to connect with.
 
+### cardano.{walletName}.supportedChains: cbor\<chain_id>[]
```

Comment:
```suggestion
### cardano.{walletName}.getSupportedChains(): Promise<\cbor\<chain_id>[]>
```
I would suggest for the sake of consistency with the rest of the API to make this an async method, also, depending on the implementation of the connector on the wallet side it may be an unnecessary complication to make this only method/property the result synchronously, especially if proxying through multiple layers (content scripts/background worker) is involved